proto/poktroll/migration/tx.proto

```diff
@@ -111,3 +114,54 @@ message MsgClaimMorseAccountResponse {
   int64 session_end_height = 3 [(gogoproto.jsontag) = "session_end_height"];
 }
 
+// TODO_UPNEXT(@bryanchriswhite, #1034):
+// - Refactor to consistent naming convention
+
+// MsgClaimMorseApplication is used to execute a claim (one-time minting of tokens on Shannon),
+// of the total tokens owned by the given Morse account, according to the on-chain MorseClaimableAccounts,
+// to the balance of the given Shannon account, followed by staking that Shannon account as an application.
+message MsgClaimMorseApplication {
+  option (cosmos.msg.v1.signer) = "shannonDestAddress";
+
+  // The bech32-encoded address of the Shannon account to which the claimed tokens
+  // will be minted and from which the application will be staked.
+  string shannonDestAddress = 1;
+
+  // The hex-encoded address of the Morse account whose balance will be claimed.
+  // E.g.: 00f9900606fa3d5c9179fc0c8513078a53a2073e
+  string morseSrcAddress = 2;
+
+  // The hex-encoded signature, by the Morse account, of this message (where this field is nil).
+  // I.e.: morse_signature = private_key.sign(marshal(MsgClaimMorseAccount{morse_signature: nil, ...}))
+  string morseSignature = 3;
+
+  // The upokt which the Shannon destination account will stake as an application.
+  cosmos.base.v1beta1.Coin stake = 4 [(gogoproto.nullable) = false];
+
+  // The services this application is staked to request service for.
+  // NOTE: This is not a repeated field, as in MsgStakeApplication,
+  // because an application can only be staked for one service.
+  string serviceConfig = 5;
```

Comment:
My understandin w.r.t this is that we aligned on having

1. Not `string serviceCOnfig` as we have here.
2. Not `repeated poktroll.shared.ApplicationServiceConfig services =` as we have in `tx.proto`
3. But `poktroll.shared.ApplicationServiceConfig services`.

If this is being done in the next PR, can you either:
1. Update the TODO_UPNEXT above (since it didn't have details)
2. Add a micro TODO here (because it's not the first time I'm getting confused)

Reply:
Again, types are updated in #1082. The intent behind separating the branches like this was to avoid including the intentional code from the generated code. I only added the comments at your previous request.